File: test/Text/test03.php
#!/usr/bin/php -c.

<?php

$srcdir = $argv[1];
$mediadir = $srcdir . "/../Media";

$m = new SWFMovie();

$f2 = new SWFFont($mediadir . "/font01.fdb");
$f3 = new SWFFont($mediadir . "/test.ttf");

$t = new SWFText(2);
$t->setFont($f2);
$t->setHeight(20);
$t->setColor(0x00, 0x00, 0x00, 0xff);
$t->moveTo(10, 20); 
$t->addString("abc");
$t->moveTo(0, 40); 
$t->addString("bca");
$t->moveTo(60, 0); 
$t->addString("cab");

$m->add($t);
$m->nextFrame(); 

$m->save("test03.swf");
?>

File: test/Text/test03.py
#!/usr/bin/python
from ming import *

import sys
mediadir = sys.argv[1]+'/../Media'

m =  SWFMovie();

f2 =  SWFFont(mediadir + "/font01.fdb")

character1 =  SWFText(2);
character1.setFont(f2);
character1.setHeight(20);
character1.setColor(0x00, 0x00, 0x00, 0xff);
character1.moveTo(10, 20);
character1.addString("abc");
character1.moveTo(0, 40);
character1.addString("bca");
character1.moveTo(60, 0);
character1.addString("cab");

i1 = m.add(character1);

m.nextFrame(); # end of frame 1 

m.save("test03.swf");
